workshops/soundboard/README.md

```diff
@@ -0,0 +1,388 @@
+# Soundboard Workshop
+
+Short link to this workshop: [https://workshops.hackclub.com/soundboard](https://workshops.hackclub.com/soundboard)
+
+---
+
+Today we're going to create our very own soundboard. You may have seen musicians such as Little Dragon, CHVRCHES, and [other musicians], using such things to improvise fresh beats at live performances. We can simulate this very easily with basic HTML and JavaScript.
+
+Much like in Personal Website workshop, we'll be using HTML to create stuff visible on the browser. We'll also be using JavaScript, a programming language commonly used in web development, to include interactivity in our project. In addition, we will be making use of jQuery, which is a library that will make using JavaScript in the browser much simpler. This may seem like an exorbitant number of new tools, but we'll be biting off manageable pieces.
+
+Table of Contents
+- Part I: Set Up
+- Part II: Connect HTML and JavaScript Files (question: should this be a subsection of Set Up)
```

Comment:
Okay, idea for subsections. How y'all feel about this?
####Part I: Set Up
**Creating the Files**
**Adding JS to the HTML File**